smart-contracts/powered-by.md

```diff
@@ -0,0 +1,131 @@
+# Dependencies Report
+
+The following is a list of all the dependencies of this project:
+## [@as-pect/cli](git+https://github.com/jtenner/as-pect.git#master)
+
+**License:** n/a - perpetual
+
+**Used version:** n/a
+
+**Many thanks to:** n/a
```

Comment:
We got no values for each section, can you try to fix it ?

Reply:
Honestly, I don't really see why these values are generated empty whereas the names are not. The script seems to handle the JSON the right way.

Reply:
Can you see if there is a difference with the repo where it works?
Don't spend more than 30 minutes on it.